packages/mdc-snackbar/mdc-snackbar.scss

```diff
@@ -21,152 +21,183 @@
 //
 
 @import "@material/animation/functions";
+@import "@material/button/mixins";
+@import "@material/icon-button/mixins";
+@import "@material/ripple/mixins";
 @import "@material/rtl/mixins";
+@import "@material/theme/functions";
```

Comment:
Are all of these theme imports used? I don't see mdc-theme anywhere in this file. I think they're used in `_mixins` instead.

Reply:
Removed